dpnp/tests/test_arraycreation.py

```diff
@@ -708,10 +708,17 @@ def test_dpctl_tensor_input(func, args):
     ids=["1", "5", "numpy.array(10)", "dpnp.array(17)", "dpt.asarray(100)"],
 )
 @pytest.mark.parametrize(
-    "dtype", get_all_dtypes(no_bool=True, no_float16=False)
+    "dtype",
+    get_all_dtypes(
+        no_bool=True, no_float16=False
+    ),
```

Comment:
```suggestion
    "dtype", get_all_dtypes(no_bool=True, no_float16=False)
```
otherwise `pre-commit` will fail

Reply:
done